tests/StatusIndicator/index.js

```diff
@@ -41,4 +41,28 @@ describe('StatusIndicator', () => {
 
     expect(wrapper.find('.status-indicator').hasClass('status-indicator--completed')).to.equal(true);
   });
+
+  it('can be the last status', () => {
+    expect(wrapper.find('.status-indicator').hasClass('status-indicator--last')).to.equal(false);
+
+    wrapper = mount(<StatusIndicator color="#00ff00" label="Review" last />);
```

Comment:
If we're just changing props, lets use `setProps` from enzyme. If this is the case then lets update all tests in this file.